server/audit_math/sampler.py

```diff
@@ -150,6 +150,7 @@ def draw_ppeb_sample(
     sample_tuples: List[Tuple[Any, Tuple[Any, Any]]] = []
 
     for batch in sample:
+        print(batch)
```

Comment:
Probably a remnant from earlier debugging that can now be removed

Reply:
I need a precommit hook that just greps for `print` :joy: